exif2timestream.py

```diff
@@ -519,7 +619,7 @@ def find_image_files(camera):
                         not dir.startswith("_"):
                     log.error("Souce directory has too many subdirs.A")
                     # TODO: Is raising here a good idea?
-                    #raise ValueError("too many subdirs")
+                    # raise ValueError("too many subdirs")
```

Comment:
Check with tim if this is valid, if it is, remove these comments, otherwise i'd go ahead and raise the error